pkg/cmd/datastore/datastore.go

```diff
@@ -418,6 +418,8 @@ func newSpannerDatastore(opts Config) (datastore.Datastore, error) {
 		spanner.WatchBufferLength(opts.WatchBufferLength),
 		spanner.EmulatorHost(opts.SpannerEmulatorHost),
 		spanner.DisableStats(opts.DisableStats),
+		spanner.ReadConnsMaxOpen(opts.ReadConnPool.MaxOpenConns),
+		spanner.WriteConnsMaxOpen(opts.WriteConnPool.MaxOpenConns),
```

Comment:
The original default was `1`, see [here](https://github.com/googleapis/google-api-go-client/blob/25093c222db5ff0fec8d97782bd65d2661f39f9c/transport/grpc/dial.go#L92-L99). 

Now we are opening 20 connections by default. If multiplied by N SpiceDB replicas and 2 pools (before 1), this could cause problems for the Spanner instance. I'd suggest setting the maximum to the original behaviour of 1, and only move it up once we understand the implications.